apps/backend/src/modules/enrollment/typedefs/enrollment.ts

```diff
@@ -12,7 +12,7 @@ export default gql`
     ): Enrollment
   }
 
-  type Enrollment @cacheControl(maxAge: 300) {
+  type Enrollment {
```

Comment:
So this max age is why it keeps refreshing the cache. Maybe it should be changed to like 15min * 60 = 900 for now? Or we can just keep as is, but we would need to put in the correct caching behavior like ASAP (otherwise enrollment numbers on catalog will be off)